test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala

```diff
@@ -191,149 +191,159 @@ class MultipleDisposalsTriageControllerSpec
       def performAction(data: (String, String)*): Future[Result] =
         controller.howManyDisposalsSubmit()(FakeRequest().withFormUrlEncodedBody(data: _*))
 
-      "redirect to disposal method when user enters number of properties as one" in {
-        val (session, journey) = sessionDataWithStartingNewDraftReturn(
-          IncompleteMultipleDisposalsAnswers.empty.copy(
-            individualUserType = Some(Self)
+      val key = "multipleDisposalsNumberOfProperties"
+
+      "redirect to disposal method" when {
```

Comment:
we're redirecting to check your answers here rather than the disposal method page